src/api/odometry/odomMath.cpp

```diff
@@ -44,4 +44,13 @@ double OdomMath::computeDistance(double xDiff, double yDiff) {
 double OdomMath::computeAngle(double xDiff, double yDiff, double theta) {
   return std::atan2(yDiff, xDiff) - theta;
 }
+
+double OdomMath::constrainAngle360(double theta) {
```

Comment:
I would rather these new methods use the units API instead of assuming that `theta` is in degrees. Then you can also avoid calling the `QAngle` constructor above.

Reply:
Changed doubles to QAngle. Not sure the unit tests are terribly elegant now, but they are serviceable.